test/unittests/iotools/converters/adni_to_bids/test_adni_utils.py

```diff
@@ -339,3 +339,26 @@ def test_get_closest_visit(closest_visit_timepoints, image_acquisition_date, exp
         ),
         closest_visit_timepoints[expected],
     )
+
+
+@pytest.fixture
+def input_df():
+    return pd.DataFrame(
+        {
+            "foo": ["foo1", "foo2", "foo3"],
+            "bar": [1, 2, 3],
+            "baz": [True, False, False],
+            "foobar": [4, 5, 6],
+        }
+    )
+
+
+@pytest.mark.parametrize(
+    "csv_name,csv_to_look_for",
+    [("adnimerge.csv", "adnimerge"), ("adnimerge_20Oct2023.csv", "adnimerge")],
+)
+def test_load_clinical_csv(tmp_path, input_df, csv_name, csv_to_look_for):
```

Comment:
Thanks for adding a test @MatthieuJoulot ! 👍

Could you also add a test verifying that the `IOError` is raised when there is no file or more than one file found.
It could also be nice to add a third test testing that the `ValueError` is raised when a single file is found but cannot be opened by Pandas (you could use another delimiter or insert a row with a wrong number of field for instance..).
WDYT?